code/game/objects/structures/signs.dm

```diff
@@ -7,26 +7,16 @@
 	w_class = ITEMSIZE_NORMAL
 
 /obj/structure/sign/ex_act(severity)
-	switch(severity)
-		if(1.0)
-			qdel(src)
-			return
-		if(2.0)
-			qdel(src)
-			return
-		if(3.0)
-			qdel(src)
-			return
-		else
-	return
+	qdel(src)
 
-/obj/structure/sign/attackby(obj/item/tool as obj, mob/user as mob)	//deconstruction
+/obj/structure/sign/attackby(obj/item/tool, mob/user)	//deconstruction
 	if(tool.isscrewdriver() && !istype(src, /obj/structure/sign/double))
-		to_chat(user, "You unfasten the sign with your [tool].")
-		unfasten()
+		playsound(get_turf(user), tool.usesound, 50, 1)
```

Comment:
I suggest using the new use_tool proc here.

Reply:
Every implementation of use_tool I've seen is for tasks that involve a `delay` time (And indeed, the proc returns immediately without a `delay` set). This action not have a delay time.